app/src/main/java/org/mozilla/focus/FocusApplication.kt

```diff
@@ -38,6 +39,8 @@ open class FocusApplication : LocaleAwareApplication(), CoroutineScope {
 
     override fun onCreate() {
         super.onCreate()
+        val exceptionsMigrator = EngineProvider.provideTrackingProtectionMigrator(this)
+        exceptionsMigrator.start(this)
```

Comment:
Let's move this after the crash reporter and log setup, so that we can see logs and crashes from it. :)

I also think this needs to go into the `isMainProcess()` block to only run it in the main process and not in every gecko child process.